tests/test_modeling_flax_common.py

```diff
@@ -178,6 +179,12 @@ def check_outputs(self, fx_outputs, pt_outputs, model_class, names):
                 Currently unused, but in the future, we could use this information to make the error message clearer
                 by giving the name(s) of the output tensor(s) with large difference(s) between PT and Flax.
         """
+        # `bigbird_block_sparse_attention` in `FlaxBigBird` returns `attention_probs = None`, while in PyTorch version,
+        # an effort was done to return `attention_probs` (yet to be verified).
+        if type(names) == str and names.startswith("attentions"):
```

Comment:
This is a bit too hacky for me here. Can't we just overwrite the test in `test_modeling_flax_big_bird.py`?